src/pwa/stores/base.js

```diff
@@ -1,10 +1,31 @@
 import { types, getParent, getEnv } from 'mobx-state-tree';
 
-export default types.model('Organization').actions(self => ({
-  addProcessors() {
-    const { h2r } = getParent(self, 1).theme;
-    const { processors } = getEnv(self).organization;
-
-    processors.forEach(proc => h2r.addProcessor(proc, 'low'));
-  },
-}));
+export default types.model('Organization').actions(self => {
+  const root = getParent(self, 1);
+
+  return {
+    addProcessors() {
+      const { h2r } = root.theme;
+      const { processors } = getEnv(self).organization;
+
+      processors.forEach(proc => h2r.addProcessor(proc, 'low'));
+    },
+    scrollToHeader() {
+      const { initialUrl } = root.build;
+      const [, page] = initialUrl.match(/\/(\d+)$/) || [null, null];
+
+      if (page) {
+        const { type, id } = root.connection.selectedItem;
+        const element = window.document.querySelector(
+          `h3[data-pagination-id='${type}_${id}_${page}']`,
+        );
+
+        if (element) {
+          // Calculates the scroll amount needed: distance from top minus navbar height.
+          const scrollAmount = element.getBoundingClientRect().top - 84;
+          window.scrollTo({ top: scrollAmount });
```

Comment:
I would add a `behavior: 'smooth'` because this is going to happen once React is loaded. It doesn't work on Safari but at least Android users will see "where they are going":
https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

Reply:
I didn't add that on purpose because it comes with the problem that it will fire all the pageviews till it arrives to the right one.

Reply:
@luisherranz I don't actually know if it's something word trying to fix. What do u think?

Reply:
> I didn't add that on purpose because it comes with the problem that it will fire all the pageviews till it arrives to the right one.

I thought you removed `fireOnRapidScroll` precisely for that.

Reply:
Leave it as it is then. Let's see what they say.

Reply:
I did it initially, but as it didn't work, I left it removed because it was pointless to define something that behaved the same way that if it wasn't there 😅